api/src/opentrons/server/endpoints/calibration/check.py

```diff
@@ -59,8 +79,10 @@ def _format_status(
         response = _format_status(new_session, request.app.router)
         return web.json_response(text=response.json(), status=201)
     else:
-        response = _format_status(current_session, request.app.router)
-        return web.json_response(text=response.json(), status=200)
+        response = {
+            "message": "A check session exists. Please delete to proceed.",
+            "links": {"DELETE": "/calibration/check/session"}}
```

Comment:
```suggestion
            "links": {"deleteSession": f"/calibration/{session_type}/session"}}
```
same thing with the verb as above